### ISSUE TITLE
Deprecation warning caused by null cpTrigger value

### ISSUE THREAD

**Issue by justinmacleod** [putyourlightson/craft-blitz#264]
### Describe the bug
There is a deprecation warning regarding strpos and non-string values. The error is being thrown in `src/services/CacheRequestService.php`. This is happening because we run our admin panels on a sub domain and the cpTrigger setting is set to null. Null is valid as per the Craft [docs](https://craftcms.com/docs/3.x/config/config-settings.html#cptrigger).

### Stacktrace

```
yii\base\ErrorException: strpos(): Non-string needles will be interpreted as strings in the future. Use an explicit chr() call to preserve the current behavior in /mnt/data/Sites/oasis/vendor/putyourlightson/craft-blitz/src/services/CacheRequestService.php:173
Stack trace:
#0 /mnt/data/Sites/oasis/vendor/craftcms/cms/src/web/ErrorHandler.php(76): yii\base\ErrorHandler->handleError()
#1 [internal function]: craft\web\ErrorHandler->handleError()
#2 /mnt/data/Sites/oasis/vendor/putyourlightson/craft-blitz/src/services/CacheRequestService.php(173): strpos()
#3 /mnt/data/Sites/oasis/vendor/putyourlightson/craft-blitz/src/Blitz.php(226): putyourlightson\blitz\services\CacheRequestService->getIsCacheableSiteUri()
#4 /mnt/data/Sites/oasis/vendor/putyourlightson/craft-blitz/src/Blitz.php(97): putyourlightson\blitz\Blitz->_registerCacheableRequestEvents()
#5 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/base/BaseObject.php(109): putyourlightson\blitz\Blitz->init()
#6 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/base/Module.php(158): yii\base\BaseObject->__construct()
#7 /mnt/data/Sites/oasis/vendor/craftcms/cms/src/base/Plugin.php(115): yii\base\Module->__construct()
#8 [internal function]: craft\base\Plugin->__construct()
#9 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/di/Container.php(400): ReflectionClass->newInstanceArgs()
#10 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/di/Container.php(159): yii\di\Container->build()
#11 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get()
#12 /mnt/data/Sites/oasis/vendor/craftcms/cms/src/services/Plugins.php(908): yii\BaseYii::createObject()
#13 /mnt/data/Sites/oasis/vendor/craftcms/cms/src/services/Plugins.php(227): craft\services\Plugins->createPlugin()
#14 /mnt/data/Sites/oasis/vendor/craftcms/cms/src/base/ApplicationTrait.php(1416): craft\services\Plugins->loadPlugins()
#15 /mnt/data/Sites/oasis/vendor/craftcms/cms/src/web/Application.php(104): craft\web\Application->_postInit()
#16 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\web\Application->init()
#17 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/base/Application.php(206): yii\base\BaseObject->__construct()
#18 /mnt/data/Sites/oasis/vendor/craftcms/cms/src/web/Application.php(92): yii\base\Application->__construct()
#19 [internal function]: craft\web\Application->__construct()
#20 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/di/Container.php(400): ReflectionClass->newInstanceArgs()
#21 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/di/Container.php(159): yii\di\Container->build()
#22 /mnt/data/Sites/oasis/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get()
#23 /mnt/data/Sites/oasis/vendor/craftcms/cms/bootstrap/bootstrap.php(246): yii\BaseYii::createObject()
#24 /mnt/data/Sites/oasis/vendor/craftcms/cms/bootstrap/web.php(51): require('/mnt/data/Sites...')
#25 /mnt/data/Sites/oasis/web/index.php(25): require('/mnt/data/Sites...')
#26 {main}
```

### Versions
- Plugin version: 7.4.11
- Craft version: 3.5.9

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-blitz/commit/77dd03c31468d317e4e198e93b7fb14ade04d6bd for the next release, thanks!

**Comment by bencroker** [comment]
Released in 3.6.12.
